### ISSUE TITLE
Big MultiGrid (>1m) scrollbar skips rows

### ISSUE THREAD

**Issue by dviry** [bvaughn/react-virtualized#800]
A big MultiGrid (seems to happen starting at 1m rows) - I can see rows 1-9 but cannot get it to show row number 10, either by clicking the scrollbar down icon or with keyboard arrow down or mousewheel. What am I missing?

On **Windows**, each Browser seems to skip a different number of rows:
Chrome Version 60.0.3112.113 (Official Build) (64-bit) - jumps to row 28
FireFox 54.0.1 (64-Bit) - jumps to row 39
Microsoft Edge 40.15063.0.0 - jumps to row 25

Basically I cannot find a way to see rows 10-24...

PS: my Windows Mouse settings are set to ONE row for the mousewheel, even then it skips multiple rows.

Here is a [Plunker sample with 1m rows](https://plnkr.co/edit/xbrrGnPY9pUHa1YkLHou) (my real case scenario actually has over 6m rows where it then skips 120 rows at once :(

*Thanks in advance for a great library!!*

**Comment by dviry** [comment]
no ideas or is no one else able to reproduce this?

or is this another "edge case" and I am not really supposed to render >1million rows with the Grid components?

Thanks

**Comment by dviry** [comment]
I am also able to reproduce this on the Demo site (obviously) in the Grid sample.

I was looking a bit more into this and it happens when 1.5m pixels height are set - after investigating a bit more the sourcecode, it's all related to the ScalingCellSizeAndPositionManager which seems already had a discussion here: #396  

Unfortunately it seems so far that ticket got closed without any changes actually being implemented... 

Maybe this could have at least been mentioned in the Docs or in the Demo somewhere... 
Closing for now as I doubt anything will ever come out of this (and it's way over my programming skills to try and do this by myself).
